pkg/kube/cluster-init.sh

```diff
@@ -29,16 +28,16 @@ setup_cgroup () {
 }
 
 check_log_file_size() {
-  currentSize=$(wc -c <"$k3slogFile")
+  currentSize=$(wc -c <"$K3S_LOG_DIR/$1")
   if [ "$currentSize" -gt "$loglimitSize" ]; then
-    if [ -f "$k3slogFile.2" ]; then
-      cp "$k3slogFile.2" "$k3slogFile.3"
+    if [ -f "$K3S_LOG_DIR/$1.2" ]; then
+      cp "$K3S_LOG_DIR/$1.2" "$K3S_LOG_DIR/$1.3"
     fi
-    if [ -f "$k3slogFile.1" ]; then
-      cp "$k3slogFile.1" "$k3slogFile.2"
+    if [ -f "$K3S_LOG_DIR/$1.1" ]; then
+      cp "$K3S_LOG_DIR/$1.1" "$K3S_LOG_DIR/$1.2"
     fi
-    cp "$k3slogFile" "$k3slogFile.1"
-    cat /dev/null > "$k3slogFile"
+    cp "$K3S_LOG_DIR/$1" "$K3S_LOG_DIR/$1.1"
+    truncate -s 0 "$K3S_LOG_DIR/$1"
```

Comment:
Why do we need that ? If log rotate is in place ?

Reply:
Oh its not real k3s based logrotate ?  Looks like poor mans logrotate :)

Reply:
Yeh, since i have already implemented that, and now it handles other logs too, such as multus.log